src/partials/our-lessons.html

```diff
@@ -0,0 +1,40 @@
+<section class="section section__lessons">
+    <div class="container container__content">
+        <div class="section__head">
+            <h2 class="section__title">
+                Our <span class="section__title--decoration">lessons</span> 
+            </h2>
+            <p class="section__content">
+                Our lessons are carefully structured and interactive, 
+                designed to promote rapid language acquisition through practical 
+                tasks and real-world scenarios
+            </p>
+        </div>
+        <ul class="lessons__list">
```

Comment:
please formatted code
We can have call for check this

Reply:
Applied formatter